### DISCUSSION TITLE
Gestion de l'affichage du calendrier indépendant de la durée de réservation minimale

### DISCUSSION THREAD

**Discussion by jicheu** [JeromeDevome/GRR#557] [Idées]
comportement actuel :
* $resolution définit la durée minimale de réservation, et aussi la granularité d'affichage du calendrier
* donc si on a un $resolution à 900 (15 minutes), on a un affichage par quart d'heure

comportement souhaité  : 
* une variable définit la durée minimale de réservation
* une autre définit la granularité d'affichage (par exemple, affichage par "heures", mais réservations de 15 minutes)

impact : 
* décorréler les deux valeurs
* modifier l'affichage (vue) qui aujourd'hui se base sur durée minimum == granularité de la "case"

**Comment by ynaessens** [comment]
Bonjour,
cela se discute.
Dans les impacts, je vois aussi que lorsqu'on cliquera sur une case, on n'aura plus le créneau de réservation prérempli.
J'attendrai de voir si cette suggestion est populaire.
Cordialement,
YN

**Comment by R4ph3uz** [comment]
Bonjour,
Cette fonctionnalité serait effectivement très intéressante, car il est regrettable d'afficher les créneaux heure par 'heure pour plus de lisibilité, sans avoir la possibilité de réserver pour seulement 30 minutes.
Merci beaucoup
